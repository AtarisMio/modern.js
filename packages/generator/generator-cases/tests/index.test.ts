import {
  getMWACases,
  getModuleCases,
  getMonorepoCases,
  getMWANewCases,
  getModuleNewCases,
  getMonorepoNewCases,
} from '../src';

describe('test generator cases', () => {
  test('test getMWACases', async () => {
    const mwaCases = getMWACases();
    expect(mwaCases).toMatchSnapshot();
  });
  test('test getModuleCases', async () => {
    const moduleCases = getModuleCases();
    expect(moduleCases).toMatchSnapshot();
  });
  test('test getMonorepoCases', async () => {
    const monorepoCases = getMonorepoCases();
    expect(monorepoCases).toMatchSnapshot();
  });
  test('test getMWANewCases', async () => {
    const mwaNewCases = getMWANewCases();
<<<<<<< HEAD
    expect(mwaNewCases.length).toBe(15);
=======
    expect(mwaNewCases.length).toBe(16);
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
  });
  test('test getModuleNewCases', async () => {
    const moduleNewCases = getModuleNewCases();
    expect(moduleNewCases).toMatchSnapshot();
  });
  test('test getMonorepoNewCases', async () => {
    const monorepoNewCases = getMonorepoNewCases();
    expect(monorepoNewCases).toMatchSnapshot();
  });
});
