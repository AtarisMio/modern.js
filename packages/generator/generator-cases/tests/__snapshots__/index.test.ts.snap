// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test generator cases test getMWACases 1`] = `
Array [
  Object {
    "clientRoute": "selfControlRoute",
<<<<<<< HEAD
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
=======
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "pnpm",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
<<<<<<< HEAD
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
=======
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "pnpm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "npm",
<<<<<<< HEAD
=======
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "pnpm",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
<<<<<<< HEAD
    "packageManager": "npm",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
=======
    "packageManager": "pnpm",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "pnpm",
<<<<<<< HEAD
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
=======
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "npm",
<<<<<<< HEAD
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
=======
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "yarn",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageManager": "npm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "yarn",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
<<<<<<< HEAD
=======
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
<<<<<<< HEAD
    "packageManager": "npm",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
=======
    "packageManager": "yarn",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "pnpm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageManager": "npm",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageManager": "npm",
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
]
`;

exports[`test generator cases test getModuleCases 1`] = `
Array [
  Object {
    "language": "js",
    "packageManager": "yarn",
    "solution": "module",
  },
  Object {
    "language": "js",
    "packageManager": "npm",
    "solution": "module",
  },
  Object {
    "language": "ts",
    "packageManager": "yarn",
    "solution": "module",
  },
  Object {
    "language": "ts",
    "packageManager": "pnpm",
    "solution": "module",
  },
  Object {
    "language": "js",
    "packageManager": "pnpm",
    "solution": "module",
  },
  Object {
    "language": "ts",
    "packageManager": "npm",
    "solution": "module",
  },
]
`;

exports[`test generator cases test getModuleNewCases 1`] = `
Array [
  Object {
    "actionType": "function",
    "function": "tailwindcss",
  },
  Object {
    "actionType": "function",
    "function": "less",
  },
  Object {
    "actionType": "function",
    "function": "sass",
  },
  Object {
    "actionType": "function",
    "function": "storybook",
  },
  Object {
    "actionType": "function",
    "function": "runtimeApi",
  },
]
`;

exports[`test generator cases test getMonorepoCases 1`] = `
Array [
  Object {
    "packageManager": "pnpm",
    "solution": "monorepo",
  },
  Object {
    "packageManager": "yarn",
    "solution": "monorepo",
  },
]
`;

exports[`test generator cases test getMonorepoNewCases 1`] = `
Array [
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
<<<<<<< HEAD
    "packageName": "ts-no-selfControlRoute",
    "packagePath": "ts-no-selfControlRoute",
=======
    "packageName": "ts-no-no-selfControlRoute",
    "packagePath": "ts-no-no-selfControlRoute",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
<<<<<<< HEAD
    "packageName": "js-yes-conventionalRoute",
    "packagePath": "js-yes-conventionalRoute",
=======
    "packageName": "js-yes-no-conventionalRoute",
    "packagePath": "js-yes-no-conventionalRoute",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
<<<<<<< HEAD
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-selfControlRoute",
    "packagePath": "ts-yes-selfControlRoute",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageName": "js-no-selfControlRoute",
    "packagePath": "js-no-selfControlRoute",
=======
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-electron-selfControlRoute",
    "packagePath": "js-yes-electron-selfControlRoute",
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
<<<<<<< HEAD
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-conventionalRoute",
    "packagePath": "ts-yes-conventionalRoute",
=======
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-electron-conventionalRoute",
    "packagePath": "js-yes-electron-conventionalRoute",
    "runWay": "electron",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
<<<<<<< HEAD
    "packageName": "js-yes-selfControlRoute",
    "packagePath": "js-yes-selfControlRoute",
=======
    "packageName": "js-yes-no-selfControlRoute",
    "packagePath": "js-yes-no-selfControlRoute",
    "runWay": "no",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
<<<<<<< HEAD
    "packageName": "ts-no-selfControlRoute",
    "packagePath": "ts-no-selfControlRoute",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-conventionalRoute",
    "packagePath": "js-yes-conventionalRoute",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-selfControlRoute",
    "packagePath": "ts-yes-selfControlRoute",
    "solution": "mwa_test",
=======
    "packageName": "ts-no-electron-selfControlRoute",
    "packagePath": "ts-no-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-electron-conventionalRoute",
    "packagePath": "ts-yes-electron-conventionalRoute",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-no-conventionalRoute",
    "packagePath": "ts-yes-no-conventionalRoute",
    "runWay": "no",
    "solution": "mwa",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
<<<<<<< HEAD
    "packageName": "js-no-selfControlRoute",
    "packagePath": "js-no-selfControlRoute",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-conventionalRoute",
    "packagePath": "ts-yes-conventionalRoute",
    "solution": "mwa_test",
=======
    "packageName": "js-no-no-selfControlRoute",
    "packagePath": "js-no-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa",
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
<<<<<<< HEAD
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-selfControlRoute",
    "packagePath": "js-yes-selfControlRoute",
    "solution": "mwa_test",
  },
  Object {
    "language": "ts",
    "packageName": "ts",
    "packagePath": "ts",
    "solution": "module",
  },
  Object {
    "language": "js",
    "packageName": "js",
    "packagePath": "js",
    "solution": "module",
  },
  Object {
=======
    "needModifyMWAConfig": "no",
    "packageName": "js-no-electron-selfControlRoute",
    "packagePath": "js-no-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-no-selfControlRoute",
    "packagePath": "ts-yes-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-electron-selfControlRoute",
    "packagePath": "ts-yes-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageName": "ts-no-no-selfControlRoute",
    "packagePath": "ts-no-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-no-conventionalRoute",
    "packagePath": "js-yes-no-conventionalRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-electron-selfControlRoute",
    "packagePath": "js-yes-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-electron-conventionalRoute",
    "packagePath": "js-yes-electron-conventionalRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "yes",
    "packageName": "js-yes-no-selfControlRoute",
    "packagePath": "js-yes-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "no",
    "packageName": "ts-no-electron-selfControlRoute",
    "packagePath": "ts-no-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-electron-conventionalRoute",
    "packagePath": "ts-yes-electron-conventionalRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "conventionalRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-no-conventionalRoute",
    "packagePath": "ts-yes-no-conventionalRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageName": "js-no-no-selfControlRoute",
    "packagePath": "js-no-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "js",
    "needModifyMWAConfig": "no",
    "packageName": "js-no-electron-selfControlRoute",
    "packagePath": "js-no-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-no-selfControlRoute",
    "packagePath": "ts-yes-no-selfControlRoute",
    "runWay": "no",
    "solution": "mwa_test",
  },
  Object {
    "clientRoute": "selfControlRoute",
    "language": "ts",
    "needModifyMWAConfig": "yes",
    "packageName": "ts-yes-electron-selfControlRoute",
    "packagePath": "ts-yes-electron-selfControlRoute",
    "runWay": "electron",
    "solution": "mwa_test",
  },
  Object {
    "language": "ts",
    "packageName": "ts",
    "packagePath": "ts",
    "solution": "module",
  },
  Object {
    "language": "js",
    "packageName": "js",
    "packagePath": "js",
    "solution": "module",
  },
  Object {
>>>>>>> 4f77eb496 (feat: remove generator create project enable less and sass function (#1659))
    "language": "ts",
    "packageName": "ts",
    "packagePath": "ts",
    "solution": "inner_module",
  },
  Object {
    "language": "js",
    "packageName": "js",
    "packagePath": "js",
    "solution": "inner_module",
  },
]
`;
